src/main/resources/category/java/remoting.xml

```diff
@@ -808,7 +808,7 @@ class Good {
             When this HttpHost is then used for a route and stored socketConfig for, port 80 is added for the host and
             the socketConfig is stored with the wrong key and will not be used.
             It is typically difficult to find out if the config is actually used. Note that
-            http://github.com/jborgers/http-client-monitor helps here.&#13;
```

Comment:
Might also be good to check and point to micrometer and available apache http 4 and 5 metrics (works with spring boot actuator).

Reply:
okay, good point, added